payment-channels-intro-lightning-htlcs/README.md

```diff
@@ -15,6 +17,13 @@
 [Hashed Timelock Contracts with Rusty Russell](https://rusty.ozlabs.org/?p=462) | 5 min | HTLCs |
 [Visualizing HTLCs and the Lightning Network's Dirty Little Secret](https://medium.com/@peter_r/visualizing-htlcs-and-the-lightning-networks-dirty-little-secret-cb9b5773a0) | 10 min | HTLCs |
 [HTLC implementation in the Core wallet](https://github.com/bitcoin/bitcoin/pull/7601) | 20 min | HTLCs |
+[Hack a LAPP tutorial by Rene Pickhardt](https://www.youtube.com/watch?v=HXVDwRnU7_I) | 6 min | c lightning LAPP hacking with python, video |
+[Making a Lightning Web App by Will O'Beirne](https://medium.com/@wbobeirne/making-a-lightning-web-app-part-1-4a13c82f3f78) | 15 min | web development on top of lnd, blog article|
+[200 Slides Explaining the BOLTs and lightning rfc](https://commons.wikimedia.org/wiki/File:Introduction_to_the_Lightning_Network_Protocol_and_the_Basics_of_Lightning_Technology_(BOLT_aka_Lightning-rfc).pdf) | 60 min | slide deck PDF |
+[Lightning Network rfc - BOLT (Basics of Lightning technologies)](https://github.com/lightningnetwork/lightning-rfc) |days | official protocol specification |
```

Comment:
This doesn't seem to fit given how long and unwieldy the BOLTs would be for a time-boxed study session. I think it would reasonable to link to them for the curious but if you imagine this is a book club, for the next chapter assignment you've appended another whole book.